openhands/tools/str_replace_editor/definition.py

```diff
@@ -199,19 +237,49 @@ class FileEditorTool(Tool[StrReplaceEditorAction, StrReplaceEditorObservation]):
     """A Tool subclass that automatically initializes a FileEditorExecutor."""
 
     @classmethod
-    def create(cls, workspace_root: str | None = None) -> "FileEditorTool":
-        """Initialize FileEditorTool with a FileEditorExecutor."""
+    def create(
+        cls,
+        workspace_root: str | None = None,
+        workspace_mount_path_in_sandbox: str | None = None,
```

Comment:
I think we won't have the concept of "in sandbox" in the context of agent-SDK (since everything runs "locally" -- we will wrap the entire SDK into docker when needed). Can we just use "workspace_root" for this dynamic path? this will be the path we will using going forward -- and will hopefully simplify the code here a ton